public/styles/common.scss

```diff
@@ -808,6 +808,13 @@ md-switch.md-checked .md-thumb {
   display: block;
 }
 
+.wz-ellipsis{
+  white-space: nowrap !important;
+  overflow: hidden !important;
+  text-overflow: ellipsis !important;
+  min-width: 0;
```

Comment:
when creating very generic classes I would be mindful of adding specific attributes that may affect others in an unexpected way.

Reply:
Right, those were adding during testing and they slipped my mind when pushing the commit. Removed them in the last one, thank you!